packages/remix/test/integration/test/server/action.test.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  assertSentryTransaction,
+  getEnvelopeRequest,
+  runServer,
+  getMultipleEnvelopeRequest,
+  assertSentryEvent,
+} from './utils/helpers';
+
+jest.spyOn(console, 'error').mockImplementation();
+
+describe('Remix API Actions', () => {
+  it('correctly instruments a parameterized Remix API action', async () => {
+    const baseURL = await runServer();
+    const url = `${baseURL}/action-json-response/123123`;
+    const envelope = await getEnvelopeRequest(url, 'post');
+    const transaction = envelope[2];
+
+    assertSentryTransaction(transaction, {
+      spans: [
+        {
+          description: 'routes/action-json-response/$id',
```

Comment:
We should really think about if we can make this name better.

Reply:
Maybe something like \`${action | loader | documentRequest} of ${route}\`?

Reply:
Yeah I like that, even if the info is a bit redundant